### DISCUSSION TITLE
Risk context

### DISCUSSION THREAD

**Discussion by dbarzin** [monarc-project/MonarcAppFO#464] [Q&A]
Hello,

There is a new field in the risk sheet called "Risk context" :

![image](https://user-images.githubusercontent.com/43208232/196349258-2a4b9d74-c66d-48af-a8b2-7c10b9cf8aec.png)

It is not described in the latest version of the documentation.  What do you expect to see in this field ?

Thanks,
Didier

**Comment by ruslanbaidan** [comment]
Hello,
This is a free text field to add more information, context or reference to the risk.
The value is only informative and not presented in any report.

**Comment by dbarzin** [comment]
Thanks for the info.
Didier
